library/styleguideComponents/Wrapper/index.js

```diff
@@ -78,6 +78,25 @@ export default function Wrapper(props) {
         return children;
     }
 
+    if (pageName === 'Customization') {
+        return (
+            <StyledWrapper>
+                <StyledTopBar>
+                    <StyledLeftElement>
+                        <InfoFilled />
+                        <StyledTitle>
+                            The theme support is <StyledHighlight>BETA</StyledHighlight>and{' '}
+                            <StyledHighlight>WIP</StyledHighlight>
+                        </StyledTitle>
+                    </StyledLeftElement>
+                </StyledTopBar>
+                <StyledContent>
+                    <Provider store={store}>{children}</Provider>
+                </StyledContent>
+            </StyledWrapper>
+        );
+    }
```

Comment:
instead of make an if clause we can simply wrap `StyledPickerTheme` component with a RenderIf with:
``` isTrue={isCustimizationPage}```